slmon/apps/slmon/slmon.py

```diff
@@ -162,16 +163,18 @@ def pageTrailer(htmlfile, config):
 
 def getColor(delta):
     delay = total_seconds(delta)
-    if   delay >432000: return '#666666'
-    if   delay >345600: return '#999999'
-    if   delay >259200: return '#cccccc'
-    if   delay >172800: return '#ffcccc'
-    if   delay > 86400: return '#ff3333'
-    elif delay > 21600: return '#ff9966'
-    elif delay >  7200: return '#ffff00'
-    elif delay >  3600: return '#00ff00'
-    elif delay >  1800: return '#3399ff'
-    else:               return '#cc99ff'
+    if   delay > 432000: return '#666666'  # > 5 days
+    elif delay > 345600: return '#999999'  # > 4 days
+    elif delay > 259200: return '#CCCCCC'  # > 3 days
+    elif delay > 172800: return '#FFB3B3'  # > 2 days
+    elif delay >  86400: return '#FF3333'  # > 1 day
+    elif delay >  21600: return '#FF9966'  # > 6 hours
+    elif delay >   7200: return '#FFFF00'  # > 2 hours
+    elif delay >   3600: return '#00FF00'  # > 1 hour
+    elif delay >   1800: return '#3399FF'  # > 30 minutes
+    elif delay >    600: return '#9470BB'  # > 10 minutes
+    elif delay >     60: return '#EBD6FF'  # >= 1 minute
```

Comment:
The comment is slightly incorrect. I should read `> 1minute` and the comment of the following line should be `<= 1 minute`.